db/rdbms-schema/src/main/resources/db/changelog/rdbms-exporter/changesets/8.8.0.xml

```diff
@@ -674,4 +674,30 @@
     </modifySql>
   </changeSet>
 
+  <changeSet id="create_correlated_message_table" author="camunda">
+    <createTable tableName="${prefix}CORRELATED_MESSAGE">
+      <column name="CORRELATION_KEY" type="VARCHAR(255)"/>
+      <column name="CORRELATION_TIME" type="TIMESTAMP WITH TIME ZONE(3)"/>
+      <column name="FLOW_NODE_ID" type="VARCHAR(255)"/>
+      <column name="FLOW_NODE_INSTANCE_KEY" type="BIGINT"/>
+      <column name="HISTORY_CLEANUP_DATE" type="TIMESTAMP WITH TIME ZONE(3)" />
+      <column name="MESSAGE_KEY" type="BIGINT">
```

Comment:
❓ This is not the message key, right ? Is a combination of 2 property. Should we name this column ? e.g message_subscription_key

Reply:
No, it's the actual message key 🙂 In RDBMS, we can simply build a composite primary key by multiple columns.